sycl/unittests/SYCL2020/DeviceGetInfoAspects.cpp

```diff
@@ -8,7 +8,7 @@
 
 #include <sycl/sycl.hpp>
 
-#include <helpers/PiImage.hpp>
+#include <helpers/MockDeviceImage.hpp>
```

Comment:
Does the test need this include?

Reply:
Removed the include in 3aa939f4dd81b176fedb6303d41d7599b4acdced